static/js/panels.js

```diff
@@ -12,17 +12,27 @@ var resize_app = function () {
 
 exports.resize_app = resize_app;
 
-var show_step = function (step) {
-    $("#panels [data-step]").hide().filter("[data-step=" + step + "]").show();
+var show_step = function ($process, step) {
+    $process.find("[data-step]").hide().filter("[data-step=" + step + "]").show();
 };
 
-var get_step = function () {
-    return $("#panels [data-step]").filter(":visible").data("step");
+var get_step = function ($process) {
+    return $process.find("[data-step]").filter(":visible").data("step");
 };
 
-exports.initialize = function () {
+exports.initialize = function ($process) {
     var ls = localstorage();
 
+    var red_notifications_list = ["email-server"];
+    // change the background color to warning-red for particular processes.
+    if (red_notifications_list.indexOf($process.data("process")) > -1) {
```

Comment:
How about marking the `<div data-process="email-server">` element as red right in the template? The information about which are supposed to be red vs. green feels like it belongs next to the text, rather than this code, and it'd also be much simpler to just write an attribute there.

The CSS selectors would have to be a little different in order to work that way, but I hope that version wouldn't be any harder to write.